functions/__fisher_path_update.fish

```diff
@@ -5,7 +5,7 @@ function __fisher_path_update -a path
         git rebase --abort
         git fetch --quiet origin master
         git reset --quiet --hard FETCH_HEAD
-        git clean --quiet -d --force
+        git clean --quiet -dx --force
```

Comment:
I see. With `-x` we basically make the repo the same as whatever was upstream. Correct?